Formula/omega-rpg.rb

```diff
@@ -41,6 +43,7 @@ def install
 
   def post_install
     # omega refuses to run without license.txt in OMEGALIB
-    ln_s prefix/"license.txt", libexec/"license.txt"
+    license_file = libexec/"license.txt"
+    prefix.install_symlink license_file unless license_file.exist?
```

Comment:
Note that this doesn't look right. We are now creating a symlink to non-existent file as symlink is in opposite direction of original.

Should be fixed in separate PR to properly post-install for existing macOS bottles, maybe something like:
```suggestion
    libexec.install_symlink prefix/"license.txt" unless (libexec/"license.txt").exist?
```